eZ/Publish/API/Repository/Tests/FieldType/RelationIntegrationTest.php

```diff
@@ -109,7 +113,7 @@ public function getValidatorSchema()
      */
     public function getValidFieldSettings()
     {
-        return array('selectionMethod' => 0, 'selectionRoot' => '1');
+        return array('selectionMethod' => 0, 'selectionRoot' => '1', 'selectionContentTypes' => array());
```

Comment:
> And here and other places where you add arrays, use PHP 5.4+ format [].

I used the old syntax cause it is used everywhere in the file, well every file I touched for this changes... didnt want to mix the style.